pkg/cri/server/container_events.go

```diff
@@ -0,0 +1,31 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package server
+
+import (
+	runtime "k8s.io/cri-api/pkg/apis/runtime/v1"
+)
+
+func (c *criService) GetContainerEvents(r *runtime.GetEventsRequest, s runtime.RuntimeService_GetContainerEventsServer) error {
+	// TODO: replace with a real implementation that broadcasts containerEventsChan to all subscribers.
```

Comment:
What's the status of the TODOs in this PR?  Are you planning to address them prior to this PR being ready to merge or do you intend to track their completion somewhere else?

Reply:
This todo is NOT a blocker of this PR because in most, if not all, use cases, there will be only one client calling GetContainerEvents. 

I can create a separate containerd issue and reference that issue in the code if that's a better way to use TODOs?

Reply:
Yes, please.

Reply:
Done. https://github.com/containerd/containerd/pull/7073/commits/281b22f43a81731a22beba402f3a3618f2cd822c#diff-69fd78c71bbb81e8ad5ce2c58609d73579d8a285ccd90056d027e49e7c21120dR24